### DISCUSSION TITLE
When the mouse clicks on webview2, contextmenustrip cannot close automatically

### DISCUSSION THREAD

**Discussion by cdz1980** [MicrosoftEdge/WebView2Feedback#3952] [Q&A]
When the mouse clicks on webview2, contextmenustrip cannot close automatically

![image](https://user-images.githubusercontent.com/59635832/164631319-679366e6-8617-49c6-9b98-b1ec1c7209b3.png)

**Comment by victorthoang** [comment]
Hello, I was wondering if you could provide some additional information about the WebView you're using (e.g. SDK and Runtime). The framework you're using would also be helpful in identifying areas we might be able to address!

If possible, could you provide a sample of your code to possibly repo and examine the issue? Thanks!

**Comment by cdz1980** [comment]
[WebView2Samples-master.zip](https://github.com/MicrosoftEdge/WebView2Feedback/files/8559590/WebView2Samples-master.zip)

Cefsharp can use CEF Domessageloopwork() sends a message to the main form to make the autoclose of contextmenustrip effective

Is there any way to turn off contextmenustrip automatically after the mouse enters webview2
